game-core/src/main/java/games/strategy/engine/display/IDisplay.java

```diff
@@ -8,10 +8,5 @@
  * on a single vm, and conversly a display may interact with many IGamePlayers
  */
 public interface IDisplay extends IChannelSubscribor {
-  /**
-   * before recieving messages, this method will be called by the game engine.
-   */
-  void initialize(IDisplayBridge bridge);
```

Comment:
`IDisplay` is a remote interface.  I don't think you can remove this method without breaking RMI compatibility.  I tried running a network game with a 3635 client, and I'm getting a ton of RMI-related errors, the first of which is

```
error in call:Remote method call, method name:shutDown remote name:games.strategy.engine.framework.AbstractGame.DISPLAY_CHANNEL
java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at games.strategy.engine.message.unifiedmessenger.EndPoint.invokeSingle(EndPoint.java:157)
	at games.strategy.engine.message.unifiedmessenger.EndPoint.invokeMultiple(EndPoint.java:140)
	at games.strategy.engine.message.unifiedmessenger.EndPoint.invokeLocal(EndPoint.java:126)
	at games.strategy.engine.message.unifiedmessenger.UnifiedMessenger.lambda$2(UnifiedMessenger.java:279)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

Reply:
@ssoloff Sorry, I missed that should we just revert this change or simply add a default body in the interface?

Reply:
I think a default interface method will work.